.issue_template/hardhat.config.ts

```diff
@@ -36,6 +39,11 @@ if (!infuraApiKey) {
   throw new Error("Please set your INFURA_API_KEY in a .env file");
 }
 
+const etherscanKey: string | undefined = process.env.ETHERSCAN_KEY;
+if (!etherscanKey) {
+  throw new Error("Please set your ETHERSCAN_KEY in a .env file");
```

Comment:
I like to put keys in `.secret` files, as it makes them a lot more obvious when pushing. Too often `.env` files get pushed :S

Reply:
The template we've been using already keeps secrets in the .env file: https://github.com/paulrberg/solidity-template/blob/main/.env.example (nothing can be worse than storing the mnemonics there), so I'd argue it's the correct place, given the template. 
.env is in .gitignore so it shouldn't be pushed to git

If you feel strongly about it, I can separate .env and .secrets, but I'd prefer to make it a separate PR, to keep each PR focused on 1 thing

Reply:
It's ok, if you think it's safe, I'm happy to let it be.

Reply:
It's ok, if you think it's safe, I'm happy to let it be.